File: ContosoWebFormsScaffolding/Samples.CS/1_Simple/Movie.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Samples.Simple
{
    public class Movie
    {
        public int Id { get; set; }

        [Required(ErrorMessage = "Movie title is required.")]
        public string Title { get; set; }

        [Required(ErrorMessage = "Movie director is required.")]
        public string Director { get; set; }

        [DataType(DataType.MultilineText)]
        public string Comments { get; set; }

        public int Count { get; set; }

        public decimal Price { get; set; }

        [Display(Name = "Release Date")]
        public DateTime ReleaseDate { get; set; }
    }

}

File: extensibilty-for-asp.net-devs/ContosoWebFormsScaffolding/Samples.VB/1_Simple/Movie.vb
Imports System.ComponentModel.DataAnnotations

Public Class Movie

    Public Property Id As Integer

    <Required(ErrorMessage:="Movie title is required.")>
    Public Property Title As String

    <Required(ErrorMessage:="Movie director is required.")>
    Public Property Director As String

    <DataType(DataType.MultilineText)>
    Public Property Comments As String

    Public Property Count As Integer

    Public Property Price As Decimal

    <Display(Name:="Release Date")>
    Public Property ReleaseDate As DateTime


End Class
